projects/angular/src/layout/breadcrumbs/breadcrumbs.html

```diff
@@ -0,0 +1,34 @@
+<div role="list" class="clr-breadcrumb-menu">
+  <ng-container *ngIf="items?.length">
+    <clr-breadcrumb-item *ngIf="items.length > max && !isExpanded">
+      <button class="btn btn-link btn-sm clr-breadcrumb-expand" (click)="expand()" (keydown)="expand()">
+        <cds-icon shape="ellipsis-horizontal"></cds-icon>
+      </button>
+    </clr-breadcrumb-item>
```

Comment:
This button needs an `aria-label`. We should add an `expandBreadcrumbsLabel` common string and use that here.